openfisca_us/tests/policy/baseline/hhs/ccdf/spm_unit_ccdf_subsidy.yaml

```diff
@@ -1,7 +1,47 @@
-- name: 1 child in DCC/SACC provider, Columbia county, NY state, family income $50,000, family size 3 
+- name: 1 child in DCC/SACC provider, Queens county, NY state, family income $50,000, family size 3 
   period: 2022
+  absolute_error_margin: 0.01
   input:
-    spm_unit_total_childcare_market_rate: 18_928
-    spm_unit_total_ccdf_copay: 2_867
+    people:
+      child:
+        ccdf_market_rate: 21_112
+        is_ccdf_eligible: true
+    spm_units:
+      spm_unit:
+        members: [child]
+        spm_unit_total_ccdf_copay: 2_804
   output:
-    spm_unit_ccdf_subsidy: 16_061
+    spm_unit_ccdf_subsidy: 18_308
+
+- name: 1 Infant in DCC/SACC provider, 1 Toddler in FDC/GFDC provider, Nassau county, NY state, Daily rates with 3 days/week for both children, family income $60,000, family size 4
+  period: 2022
+  absolute_error_margin: 0.01
+  input:
+    people:
+      child1:
+        ccdf_market_rate: 10_140
+        is_ccdf_eligible: true
+      child2:
+        ccdf_market_rate: 9_360
+        is_ccdf_eligible: true
+    spm_units:
+      spm_unit:
+        members: [child1, child2]
+        spm_unit_total_ccdf_copay: 6_850
+  output:
+    spm_unit_ccdf_subsidy: 12_650
+
+- name: 1 child in DCC/SACC provider, Queens county, NY state, family income $50,000, family size 3, family asset above $1,000,000
+  period: 2022
+  absolute_error_margin: 0.01
+  input:
+    people:
+      child:
+        ccdf_market_rate: 21_112
+        is_ccdf_eligible: false
+    spm_units:
+      spm_unit:
+        members: [child]
+        spm_unit_total_ccdf_copay: 2_804
+  output:
+    spm_unit_ccdf_subsidy: 0
```

Comment:
Could just add another test with eligible children whose copay exceeds their market rate to check the subsidy is zero? After that looks good to merge.